actionpack/lib/action_dispatch/http/mime_negotiation.rb

```diff
@@ -84,7 +84,45 @@ def formats
         end
       end
 
-      # Sets the \variant for template.
+      # Sets the \variant for the response template.
+      #
+      # Action View will incorporate the request's variants when determining which template to render. For example, if an <tt>ArticlesController</tt> needs to respond to an <tt>#index</tt> action request with <tt>request.variant = [:ios, :turbo_native]</tt>, it will render the first template file it can find in the following list:
```

Comment:
```suggestion
      # When determining which template to render, Action View will incorporate all variants from the request.
      # For example, if an <tt>ArticlesController#index</tt> action needs to respond to <tt>request.variant = [:ios, :turbo_native]</tt>, it will render the first template file it can find in the following list:
```